translator/gobgp.py

```diff
@@ -22,14 +22,31 @@ def _get_family(self, ip_version):
         else:
             return gobgp_pb2.Family.AFI_IP
 
-    def _build_path(self, ip):
+    def _build_path(self, ip, event_data):
+        # defaults
+        asn = 64500
+        community = 666
+
+        logging.debug(event_data)
```

Comment:
Might want to revisit this later. One day, the event data might have BGP authentication data, or something else that shouldn't be logged in the clear.